mls_validation_service/Cargo.toml

```diff
@@ -25,7 +25,7 @@ tracing-subscriber = { workspace = true, features = ["env-filter"] }
 tracing.workspace = true
 warp = "0.3.6"
 xmtp_cryptography = { path = "../xmtp_cryptography" }
-xmtp_id.workspace = true
+xmtp_id = { workspace = true, features = ["test-utils"] }
```

Comment:
I don't think we want this in the prod build. I added it just for debugging our issues with local dev.

Reply:
Oh, woops. I thought this was dev-dependencies. Let me remove it.